src/main/java/com/fitpet/server/user/infra/jpa/UserJpaRepository.java

```diff
@@ -45,4 +47,11 @@ Page<User> findInactiveUsers(
             @Param("inactiveSince") LocalDateTime inactiveSince,
             Pageable pageable
     );
+
+    List<User> findTop10ByOrderByDailyStepCountDescUpdatedAtAsc();
+
+    List<User> findTop10ByGenderOrderByDailyStepCountDescUpdatedAtAsc(Gender gender);
+
+    long countByDailyStepCountGreaterThan(int dailyStepCount);
```

Comment:
조건을 추가하면서 메소드명이 너무 길어져 가독성을 해치는 것 같습니다!


`@Query`를 통해 쿼리를 명시해주고 비즈니스 의미를 담아 간결하게 수정할 것을 제안드립니다🧐

추후 요구되는 랭킹의 수(예 : Top20 )가 변경 될 수도 있기 때문에 개수를 파라미터로 받으면 좋을 것 같습니다.
=> 적용시 service 쪽에도 수정해야 합니다!

```suggestion
  @Query("SELECT u FROM User u ORDER BY u.dailyStepCount DESC, u.updatedAt ASC")
      List<User> findDailyRanking(Pageable pageable);
  
    @Query("SELECT u FROM User u WHERE u.gender = :gender ORDER BY u.dailyStepCount DESC, u.updatedAt ASC")
    List<User> findGenderDailyRanking(@Param("gender") Gender gender, Pageable pageable);

```

Reply:
수정 완료했습니다 !! 
좋은 의견 감사합니다 :)